src/Models/UserModel.php

```diff
@@ -21,6 +21,7 @@
 use CodeIgniter\Shield\Exceptions\InvalidArgumentException;
 use CodeIgniter\Shield\Exceptions\ValidationException;
 use Faker\Generator;
+use LogicException;
```

Comment:
```suggestion
use CodeIgniter\Shield\Exceptions\LogicException;
```
and run `composer cs-fix`.